### ISSUE TITLE
Create Cloud Function that returns the list of trainable labels

### ISSUE THREAD

**Issue by Loenike** [Passiolife/passio_devops#45]
We need to have a function that determines the list of current labels that are "trainable".

Since the approach of this is still not fixed, we probably first build a cloud function, and once we are safe, we can translate it to an API endpoint (if needed).

That cloud function would return a list of labelnames and the function would be called:
- when James does the training (frozen training database)
- when we are exporting the SQLite database (frozen training database)
- potentially Brandon calls this function when displaying the labels inside the spreadsheet tool inside Food Data Editor (production database)

- [ ] @Loenike share the function with Nabil
- [ ] @ncheikh-passio implement the cloud funciton

**Comment by Loenike** [comment]
@kells1986 FYI

**Comment by Loenike** [comment]
@sptrakesh Do you think a cloud function is  a good solution for this here?

Asking because, we will probably need to change the thresholds / definitions on what is a "trainable" label a bit more until we have a "fixed" approach - that's why I am hesitant to have it as an API endpoint.

**Comment by sptrakesh** [comment]
This seems to be a true API endpoint, so I would add it to the appropriate project and not deploy as a function.  I see the benefit of using a function temporarily until the logic/requirements are dialled in, since it only involves deploying that function.

**Comment by Loenike** [comment]
Ok! Makes sense! Let's temporarily have the function until we are 100% sure about the approach, the we can convert to API endpoint. Thanks!

**Comment by Loenike** [comment]
@ncheikh-passio I shared the first draft here:
https://github.com/Passiolife/passio_cloud_functions/pull/9

**Comment by ncheikh-passio** [comment]
@Loenike I agree with @sptrakesh it seems like a true endpoint in the long run but a cloud function for now will let us iterate faster in the near term.
